packages/teleterm/src/ui/DocumentCluster/ClusterResources/Servers/Servers.tsx

```diff
@@ -16,15 +16,20 @@ limitations under the License.
 
 import React from 'react';
 
-import Table, { Cell } from 'design/DataTable';
+import styled from 'styled-components';
```

Comment:
nit: we don't need so many empty lines between imports, something like this is enough:
```typescript
import React from 'react';
import styled from 'styled-components';
import { Box } from 'design';
import { Danger } from 'design/Alert';
import Table, { Cell, ClickableLabelCell } from 'design/DataTable';
import { MenuLogin } from 'shared/components/MenuLogin';
import { SearchPanel } from 'shared/components/Search';
import { SearchPagination } from 'shared/components/Search/SearchPagination';

import * as types from 'teleterm/ui/services/clusters/types';

import { MenuLoginTheme } from '../MenuLoginTheme';

import { useServers, State } from './useServers';
```